src/components/WwwFrame/TheHeader.vue

```diff
@@ -563,7 +564,9 @@ export default {
 			}
 		},
 		onLendMenuShow() {
```

Comment:
Fire this function on every hover. To fire track event. If control, open lend menu 'onOpen' function.